website/source/api/auth/cert/index.html.md

```diff
@@ -29,21 +29,34 @@ Sets a CA cert and associated parameters in a role name.
 
 - `name` `(string: <required>)` - The name of the certificate role.
 - `certificate` `(string: <required>)` - The PEM-format CA certificate.
-- `allowed_names` `(string: "")` - Constrain the Common and Alternative Names in 
+- `allowed_names` `(string: "")` - Constrain the Common and Alternative Names in
   the client certificate with a [globbed pattern]
-  (https://github.com/ryanuber/go-glob/blob/master/README.md#example). Value is 
-  a comma-separated list of patterns.  Authentication requires at least one Name matching at least one pattern.  If not set, defaults to allowing all names.
-- `required_extensions` `(string: "" or array:[])` - Require specific Custom Extension OIDs to exist and match the pattern. 
-   Value is a comma separated string or array of `oid:value`. Expects the extension value to be some type of ASN1 encoded string.
-   All conditions _must_ be met. Supports globbing on `value`.
-- `policies` `(string: "")` - A comma-separated list of policies to set on tokens 
-  issued when authenticating against this CA certificate.
-- `display_name` `(string: "")` -   The `display_name` to set on tokens issued 
-  when authenticating against this CA certificate. If not set, defaults to the 
+  (https://github.com/ryanuber/go-glob/blob/master/README.md#example). Value is
+  a comma-separated list of patterns. Authentication requires at least one Name
+  matching at least one pattern. If not set, defaults to allowing all names.
+- `required_extensions` `(string: "" or array:[])` - Require specific Custom
+   Extension OIDs to exist and match the pattern. Value is a comma separated
+   string or array of `oid:value`. Expects the extension value to be some type
+   of ASN1 encoded string. All conditions _must_ be met. Supports globbing on
+   `value`.
+- `policies` `(string: "")` - A comma-separated list of policies to set on
+  tokens issued when authenticating against this CA certificate.
+- `display_name` `(string: "")` - The `display_name` to set on tokens issued
+  when authenticating against this CA certificate. If not set, defaults to the
   name of the role.
-- `ttl` `(string: "")` - The TTL period of the token, provided as a number of 
-  seconds. If not provided, the token is valid for the the mount or system 
+- `ttl` `(string: "")` - The TTL period of the token, provided as a number of
+  seconds. If not provided, the token is valid for the the mount or system
   default TTL time, in that order.
+- `max_ttl` `(string: "")` - Duration in either an integer number of seconds
+  (`3600`) or an integer time unit (`60m`) after which the issued token can no
+  longer be renewed.
+- `period` `(string: "")` - Duration in either an integer number of seconds
+  (3600) or an integer time unit (60m). If set, the generated token is a
```

Comment:
Integer time unit is not used anywhere else in the docs and is a little confusing.  I've seen a duration with hours as the largest suffix as well as some other ones.  We should probably make the docs consistent for ttl, max_ttl, and period.